MahApps.Metro/Themes/NumericUpDown.xaml

```diff
@@ -68,8 +68,31 @@
                                 <ColumnDefinition x:Name="PART_NumericDownColumn"
                                                   Width="Auto" />
                             </Grid.ColumnDefinitions>
-
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto"></RowDefinition>
+                                <RowDefinition Height="*"></RowDefinition>
+                            </Grid.RowDefinitions>
+                            <Grid x:Name="PART_FloatingMessageContainer"
+                                  Grid.Column="0"
+                                  Height="0"
+                                  IsHitTestVisible="False"
+                                  Margin="5,0"
+                                  Visibility="Visible">
+                                <TextBlock x:Name="PART_FloatingMessage"
+                                           Text="{TemplateBinding Controls:TextBoxHelper.Watermark}"
+                                           FontSize="{DynamicResource FloatingWatermarkFontSize}"
+                                           Foreground="{TemplateBinding Foreground}"
+                                           Opacity="0.6"
+                                           HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+                                           VerticalAlignment="{TemplateBinding VerticalContentAlignment}">
+                                    <TextBlock.RenderTransform>
+                                        <TranslateTransform />
+                                    </TextBlock.RenderTransform>
+                                </TextBlock>
+                            </Grid>
                             <TextBox x:Name="PART_TextBox"
```

Comment:
@xxMUROxx can you change the floating message part like in https://github.com/MahApps/MahApps.Metro/commit/1919458bce35ba5c5815e2c9982721836263cdb2 ?
thx